src/index.js

```diff
@@ -0,0 +1,279 @@
+import './style.css';
+
+// Create image for more options
+import Vert from '../icons/vert.png';
+
+// Create trashcan image
+import TrashCan from '../icons/trashcan.png';
+
+// Create ToDoTasks object
+import ToDoTasks from './modules/tasks.js';
+
+const tasks = new ToDoTasks();
+
+if (localStorage.getItem('data') !== null) {
+  // If there is data stored, set collection to that data
+  tasks.list = JSON.parse(localStorage.getItem('data'));
+}
+
+const list = document.querySelector('ul');
+
+// Updates the indexes of each element
+function updateDomIndexes() {
+  const parent = document.querySelector('ul');
+  const elements = parent.querySelectorAll('li');
+  for (let i = 0; i < elements.length; i += 1) {
+    elements[i].className = i + 1;
+  }
+}
+
```

Comment:
### Exporting/importing classes and functions as modules
Following the ES6 and Javascript best practices, it would be better if you export/import your functions and put them in separate files. This will make your code look more orderly, and we can make changes only in a certain section without affecting the rest of the code, as well as make it easier to apply testing techniques later in the process.